src/Serializer/V3/MatchingContextNormalizer.php

```diff
@@ -55,12 +55,16 @@ public function normalize($matchingContext, $format = null, array $context = [])
 
         return array_filter([
             'id' => $matchingContext->getId(),
+            'domains' => $matchingContext->getDomains(),
             'noticeId' => $matchingContext->getNotice()->getId(),
             'noticeUrl' => $this->router->generate(
                 'app_api_v3_getnoticeaction__invoke',
                 ['id' => $matchingContext->getNotice()->getId()],
                 RouterInterface::ABSOLUTE_URL),
+            // @todo should be renamed `fullUrlRegex` in next major release
             'urlRegex' => $matchingContext->getFullUrlRegex($this->escaper),
+            // @todo should probably be named `urlRegex` in next major release
+            'urlPathRegex' => $matchingContext->getEscapedUrlRegex($this->escaper),
```

Comment:
Why not just `-> getUrlRegex()` ?  Since we don’t escape this part in `->getFullUrlRegex($this->escaper)` … ?

Reply:
Alright, thought this part was escaped too.

Reply:
It seems not, if you look at `getFullUrlRegex` : `return '('.implode('|', $domains).')'.$this->urlRegex;`